Mimir/src/helpers/onlineLog/Parser.php

```diff
@@ -67,11 +69,35 @@ public function parseToSession(SessionPrimitive $session, string $content, $with
         $reader = new \XMLReader();
         $reader->XML($content);
 
+        $kanNakiCache = null;
+        $this->_ankan_cache = array(
+            0 => array(),
+            1 => array(),
+            2 => array(),
+            3 => array());
```

Comment:
1) please use camel case
2) `array()` can be replaced by `[]`, please use `[]` for consistency with other codebase.
3) array indices can be omitted when starting from 0
```
$this->_ankanCache = [[], [], [], []];
```

Reply:
fixed